apps/explorer/src/ui/DateCard.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Text } from '~/ui/Text';
+import { formatDate } from '~/utils/timeUtils';
+
+export interface DateCardProps {
+    date: Date | number;
+}
+
+// TODO - add format options
+export function DateCard({ date }: DateCardProps) {
+    const dateStr = formatDate(date, [
+        'month',
+        'day',
+        'year',
+        'hour',
+        'minute',
+    ]);
+    return dateStr && dateStr !== '' ? (
+        <div className="text-sui-grey-75">
+            <Text variant="bodySmall" weight="semibold">
+                <time dateTime={dateStr}>{dateStr}</time>
```

Comment:
The `dateTime` attribute needs a specific format, you can construct a date object and call `toISOString()`.

Reply:
done